milatools/cli/commands.py

```diff
@@ -383,6 +383,10 @@ def mila():
     return function(**args_dict)
 
 
+def test_uncovered_code():
+    ...
+
+
```

Comment:
This should have a comment mentioning that it isn't covered. We'll have to wait until the codecov app is approved by IT to get them.